cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -131,21 +137,20 @@ friendlyTxOut (TxOut addr amount mdatum) =
           datum ShelleyBasedEraAllegra = []
           datum ShelleyBasedEraMary = []
           datum ShelleyBasedEraAlonzo = ["datum" .= renderDatum mdatum]
-      in object $ preAlonzo ++ datum sbe
+      in preAlonzo ++ datum sbe
   where
-   renderDatum :: TxOutDatum CtxTx era -> Aeson.Value
-   renderDatum TxOutDatumNone = Aeson.Null
-   renderDatum (TxOutDatumHash _ h) =
-     Aeson.String $ serialiseToRawBytesHexText h
-   renderDatum (TxOutDatum _ sData) =
-     scriptDataToJson ScriptDataJsonDetailedSchema sData
+    renderDatum :: TxOutDatum CtxTx era -> Aeson.Value
```

Comment:
> * Indent the `where` keyword with one space to set it
apart from the rest of the code and indent the definitions in a
`where` clause with 1 space.

Reply:
I find it unnatural, but done